packages/create-project/test/cli_test.js

```diff
@@ -1,23 +1,48 @@
 import test from 'ava';
-import { xprod } from 'ramda';
 import assert from 'yeoman-assert';
 import helpers from 'yeoman-test';
 import { join } from 'path';
 import { spawn } from 'child_process';
-import { packages } from '../commands/init/utils'
-
-if (process.env.NODE_ENV !== 'test') {
-  process.env.NODE_ENV = 'test';
```

Comment:
I'm guessing this isn't needed since Neutrino sets `NODE_ENV` appropriately anyway?